### PR TITLE
EZP-25546: Pointer events block scrolling after loading is finished in Google Chrome

### PR DESCRIPTION
**Pull request by sunpietro**
https://jira.ez.no/browse/EZP-25546

Explicitly set `pointer-events` to `all` when the app is opened but in loading state.

### CODE REVIEW

**Comment by sunpietro** [comment]
ping @dpobel

**Comment by andrerom** [comment]
+1  (bug fix 1.2.x branch)

**Comment by lserwatka** [comment]
@sunpietro with this patch we don't have to fix anything on the studio side, correct?

**Comment by sunpietro** [comment]
@lserwatka yes, this patch fixes the blocker EZS-605 in eZ Studio.

**Comment by sunpietro** [comment]
I found that, my proposal doesn't fix the issue on Safari. To fix scrolling issue there, I have to remove the `pointer-events: none;` property completely.

**Comment by sunpietro** [comment]
Created a different approach only for Studio app: ezsystems/StudioUIBundle#462

**Comment by lserwatka** [comment]
@andrerom @dpobel We have to make a decision soon. Either we fix it here or in StudioUI. What I understood, that fixing here (with proposed patch) is not enough for Safari browser. I'm more in favor to have a fix in StudioUI and create 1.2.1 tag there (fix for master and 1.2.x).

**Comment by dpobel** [comment]
so I understand correctly, this patch fixes the issue in Chrome but not in Safari ? Did you try with `pointer-events: auto;` instead of `pointer-events: all;` ?

**Comment by dpobel** [comment]
(according to https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events it seems `all` is dedicated to SVG element)

**Comment by sunpietro** [comment]
@dpobel yes, I did
we could either remove the `pointer-events: none;` or override it like I did in the PR for Studio

**Comment by lserwatka** [comment]
I can see one risk of not fixing it here. Other plugin devs might experience same issues.

**Comment by dpobel** [comment]
I meant testing in this patch `pointer-events: auto;` (instead `of all`).

File: Resources/public/css/layout.css [hunk 1]
```diff
@@ -29,6 +29,7 @@
     opacity: 1;
     min-height: 100%;
     overflow: auto;
+    pointer-events: all;
```
Comment (dpobel) [inline Resources/public/css/layout.css]: I would put it on `.ez-platformui-app`

Comment (sunpietro) [inline Resources/public/css/layout.css]: Well, where is it placed doesn't really matter. The `is-app-open` CSS class is added as soon as the Loading application screen disappears.

We can keep the `.is-app-loading` selector alone without `.is-app-open` at the beggining of the selector.
I did it this way to show, that `.is-app-loading` class is added when the app is opened but switching active views.

To sumarize, I'd keep it here.

Comment (dpobel) [inline Resources/public/css/layout.css]: > Well, where is it placed doesn't really matter. The is-app-open CSS class is added as soon as the Loading application screen disappears.

for you maybe, for me (as a maintainer of PlatformUI) it matters. The opposite of 'is-app-loading' is not having 'is-app-loading', not 'is-app-open' which is a different concept.

> We can keep the .is-app-loading selector alone without .is-app-open at the beggining of the selector.

so please do NOT add unneeded changes. A bug fix is a fix with the minimum amount of change.

> I did it this way to show, that .is-app-loading class is added when the app is opened but switching active views.

which is an implementation detail irrelevant to the problem. *open* and *loading* are 2 different concepts which are not really related.

> To sumarize, I'd keep it here as it is.

I prefer to not answer to that...

**Comment by sunpietro** [comment]
ping @dpobel

**Comment by dpobel** [comment]
+1 (when Travis agrees)

**Comment by sunpietro** [comment]
@dpobel behat is failing for some reason

**Comment by dpobel** [comment]
@sunpietro let's ignore CI tests, I have absolutely no clue of what's going on there. Since [yesterday](https://github.com/ezsystems/PlatformUIBundle/pull/524), Travis runs the BDD tests, and there everything is up to date we can see what really happens.

**Comment by dpobel** [comment]
@sunpietro also failing Travis but it seems completely unrelated to your patch. I'm restarting the job.

@andrerom @bdunogier any clue on that https://gist.github.com/dpobel/bdfe9769e41d20117511#file-travis-log-L1633 ?

**Comment by bdunogier** [comment]
> @andrerom @bdunogier any clue on that https://gist.github.com/dpobel/bdfe9769e41d20117511#file-travis-log-L1633 ?

Really not, gotta admit... new one to me :(

**Comment by lserwatka** [comment]
@dpobel @sunpietro is this something we can merge for 1.2.1 tag?

**Comment by dpobel** [comment]
ping @StephaneDiot @yannickroger

**Comment by StephaneDiot** [comment]
+1

**Comment by miguelcleverti** [comment]
@bdunogier yes the object manager will be removed

**Comment by dpobel** [comment]
Replaced by https://github.com/ezsystems/PlatformUIBundle/pull/530

**Pull request closed without merging**